### ISSUE TITLE
fatal error: 'cpucycles.h' file not found

### ISSUE THREAD

**Issue by OR13** [gbanegas/sphss#1]
```
❯ make
Creating directories
Beginning release build
Compiling: fips180.c -> build/release/fips180.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c fips180.c -o build/release/fips180.o
	 Compile time: 1
Compiling: fips202.c -> build/release/fips202.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c fips202.c -o build/release/fips202.o
	 Compile time: 0
Compiling: hash.c -> build/release/hash.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c hash.c -o build/release/hash.o
	 Compile time: 0
Compiling: hss.c -> build/release/hss.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c hss.c -o build/release/hss.o
	 Compile time: 1
Compiling: lms.c -> build/release/lms.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c lms.c -o build/release/lms.o
	 Compile time: 0
Compiling: lms_ots.c -> build/release/lms_ots.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c lms_ots.c -o build/release/lms_ots.o
	 Compile time: 0
Compiling: lms_utils.c -> build/release/lms_utils.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c lms_utils.c -o build/release/lms_utils.o
	 Compile time: 0
Compiling: main.c -> build/release/main.o
cc  -DLMOTS_SHA256_N32_W8 -DLMS_SHA256_M32_H5 -O3 -c -fmessage-length=0 -march=native -I . -MP -MMD -c main.c -o build/release/main.o
main.c:9:10: fatal error: 'cpucycles.h' file not found
#include "cpucycles.h"
         ^~~~~~~~~~~~~
1 error generated.
make[1]: *** [build/release/main.o] Error 1
make: *** [release] Error 2

```

**Comment by gbanegas** [comment]
Sorry,  to take fix this. I will soon add the file.

**Comment by gbanegas** [comment]
This is fixed in commit 6a69c2ed4eb4d7f6ed05bdb503fdcd0d55c5b628.
